lib/src/view/play/challenge_screen.dart

```diff
@@ -56,6 +63,25 @@ class _ChallengeBody extends ConsumerStatefulWidget {
 
 class _ChallengeBodyState extends ConsumerState<_ChallengeBody> {
   Future<void>? _pendingCreateGame;
+  final _controller = TextEditingController();
+
+  String? textInput;
```

Comment:
`textInput` is too vague, prefer, for a local property, a long named describing exactly what it is representing.